client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -256,6 +256,11 @@ export const useStoreItemInfo = ( {
 		( item: SelectorProduct ) => {
 			const shouldShowCart = getShouldShowCart( item );
 			const isProductInCart = getIsProductInCart( item );
+			const isOwned = getIsOwned( item );
+
+			if ( isOwned ) {
+				return <>{ translate( 'Manage Subscription' ) }</>;
```

Comment:
I know this is the pattern in this function, I am just not sure the Fragments (<></>) are needed here? `translate` should just return a string in this scenario, maybe there's some reason I am unaware of?

Reply:
Good point, I will have that removed.

Reply:
```
if ( isProductInCart ) {
				return (
					<>
						<Gridicon icon="checkmark" />
						{ translate( 'View Cart' ) }
					</>
				);
			}
```

The same function had this bit which needs the `<React.Fragment>`, I reckon we added it to all. I have removed it where we don't need it.